parser/v2/elementparser.go

```diff
@@ -422,5 +439,18 @@ func (elementParser) Parse(pi *parse.Input) (r Element, ok bool, err error) {
 		err = parse.Error(fmt.Sprintf("<%s>: %s", r.Name, strings.Join(msgs, ", ")), start)
 	}
 
+	r.LineBreak = true
```

Comment:
It's not easy to understand what "LineBreak" means. Does it mean that the element should not be placed on a newline because it's inline and the previous node is text and ends with whitespace?

Reply:
"LineBreak" is to indicate that we must put a line break after the element.

By default we assume that every element should line break except our Inline Element.

But let me explore more `IndentChildren` after re locking your code here I have some feeling they was already something with this behavior I could just change.